pubnub-kotlin/pubnub-kotlin-impl/src/integrationTest/kotlin/com/pubnub/api/integration/PublishIntegrationTests.kt

```diff
@@ -161,6 +162,32 @@ class PublishIntegrationTests : BaseIntegrationTest() {
         }
     }
 
+    @Test
+    fun testReceiveMessageWithConfigOverride2() {
+        val expectedChannel = randomChannel()
+        val messagePayload = generateMessage(pubnub)
+        val expectedUser = PubNub.generateUUID()
+
+        val observer = createPubNub {}
+        pubnub.test {
+            subscribe(expectedChannel)
+            observer.publish(
+                message = messagePayload,
+                channel = expectedChannel,
+            ).apply {
```

Comment:
I was expecting to be able to do:
```
            observer.publish(
                message = messagePayload,
                channel = expectedChannel,
            ).overrideConfiguration(overwrittenConfiguration).sync()
```
Why not to add this possibility?

Reply:
added